sdk/contentsafety/azure-ai-contentsafety/README.md

```diff
@@ -0,0 +1,288 @@
+# Azure ContentSafety client library for Java
+
+[Azure AI Content Safety][contentsafety_overview] detects harmful user-generated and AI-generated content in applications and services. Content Safety includes several APIs that allow you to detect material that is harmful:
+
+* Text Analysis API: Scans text for sexual content, violence, hate, and self harm with multi-severity levels.
+* Image Analysis API: Scans images for sexual content, violence, hate, and self harm with multi-severity levels.
+* Text Blocklist Management APIs: The default AI classifiers are sufficient for most content safety needs; however, you might need to screen for terms that are specific to your use case. You can create blocklists of terms to use with the Text API.
+
+## Documentation
+
+Various documentation is available to help you get started
+
+- [API reference documentation][docs]
+- [Product documentation][product_documentation]
+
+## Getting started
+
+### Prerequisites
+
+- [Java Development Kit (JDK)][jdk] with version 8 or above
+- You need an [Azure subscription][azure_sub] to use this package.
+- An existing [Azure AI Content Safety][contentsafety_overview] instance.
+
+### Adding the package to your product
+
+[//]: # ({x-version-update-start;com.azure:azure-ai-contentsafety;current})
+```xml
+<dependency>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-ai-contentsafety</artifactId>
+    <version>1.0.0-beta.1</version>
+</dependency>
+```
+[//]: # ({x-version-update-end})
+
+### Authenticate the client
+
+#### Get the endpoint
+You can find the endpoint for your Azure AI Content Safety service resource using the [Azure Portal][azure_portal] or [Azure CLI][azure_cli_endpoint_lookup]:
+
+```bash
+# Get the endpoint for the Azure AI Content Safety service resource
+az cognitiveservices account show --name "resource-name" --resource-group "resource-group-name" --query "properties.endpoint"
+```
+
+#### Get the API key
+
+The API key can be found in the [Azure Portal][azure_portal] or by running the following [Azure CLI][azure_cli_key_lookup] command:
+
+```bash
+az cognitiveservices account keys list --name "<resource-name>" --resource-group "<resource-group-name>"
+```
```

Comment:
nit, you may want to make cmd consistent.

one is name with out `<>`, another is with `<>`

Reply:
resolved